var/spack/repos/builtin/packages/bitsery/package.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
```

Comment:
Is this package necessary to support the `aocc` package? I don't see it being needed based on looking at the `depends_on` declarations.

Could you remove this and other package files in `var/spack/repos/builtin/packages/` except for `aocc/package.py`?

Reply:
Yes, this will go away in my new PR https://github.com/spack/spack/pull/19345